src/main/java/org/traccar/protocol/TopinProtocolDecoder.java

```diff
@@ -60,6 +63,21 @@ private void sendResponse(Channel channel, int length, int type, ByteBuf content
         }
     }
 
+    private void updateTime(Channel channel, int msgType){
+        ByteBuf dateBuffer = Unpooled.buffer();
+
+        Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+
+        dateBuffer.writeBytes(BigInteger.valueOf(calendar.get(Calendar.YEAR)).toByteArray());
```

Comment:
Can we use `writeShort` instead of this?

Reply:
Fixed in commit 6fc7210.